src/vec.rs

```diff
@@ -42,24 +45,55 @@ impl<T: StableDeref> FrozenVec<T> {
         }
     }
 
+    /// Returns a reference to an element.
     pub fn get(&self, index: usize) -> Option<&T::Target> {
         unsafe {
             let vec = self.vec.get();
             (*vec).get(index).map(|x| &**x)
         }
     }
 
+    /// Returns a reference to an element, without doing bounds checking.
```

Comment:
issue: Please add a `## Safety` section that documents that `index` must be in bounds (less than the `len`